### PR TITLE
Part 3 v6 release: Update download script and doc

### PR DESCRIPTION
**Pull request by kgaonkar6**
🚨  Only merge this PR once #43 #44  is reviewed and merged.

#### What scientific question is your analysis addressing?
Data release v6

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/57

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
Please review documentation for release in `doc` folder

### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [x] Is doc/data-format.md up to date?
- [x] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date?
- [x] Was download-data.sh tested and did it complete without error?

### CODE REVIEW

File: doc/data-formats.md [hunk 1]
```diff
@@ -22,11 +22,21 @@ Somatic calls that are retained if they are supported by atleast 2 callers OR ma
 
 * `snv-consensus-plus-hotspots.maf.tsv.gz`
 
+### Somatic Copy Number Variant (CNV) Data
+
+Somatic Copy Number Variant (CNV) data are provided in a modified [SEG format](https://software.broadinstitute.org/software/igv/SEG) for each of the [applied software packages](https://alexslemonade.github.io/OpenPBTA-manuscript/#somatic-copy-number-variant-calling) and denoted with the `pbta-cnv` prefix.
+**Somatic copy number data is only generated for whole genome sequencing (WGS) samples.**
+
+  * `pbta-cnv-cnvkit.seg.gz` is the the CNVkit SEG file. This file contains an additional column `copy.num` to denote copy number of each segment, derived from the CNS file output of the algorithm [described here](https://cnvkit.readthedocs.io/en/stable/fileformats.html).
+  * `pbta-cnv-controlfreec.tsv.gz` is the ControlFreeC TSV file. It is a merge of `*_CNVs` files produced from the algorithm, and columns are [described here](http://boevalab.inf.ethz.ch/FREEC/tutorial.html#OUTPUT).
```
Comment (jharenza) [inline doc/data-formats.md]:
```suggestion
Somatic Copy Number Variant (CNV) data are provided in a modified [SEG format](https://software.broadinstitute.org/software/igv/SEG) for each of the [applied software packages](https://alexslemonade.github.io/OpenPBTA-manuscript/#somatic-copy-number-variant-calling) and denoted with the `cnv` prefix.
**Somatic copy number data is only generated for whole genome sequencing (WGS) samples.**

  * `cnv-cnvkit.seg.gz` is the the CNVkit SEG file. This file contains an additional column `copy.num` to denote copy number of each segment, derived from the CNS file output of the algorithm [described here](https://cnvkit.readthedocs.io/en/stable/fileformats.html).
  * `cnv-controlfreec.tsv.gz` is the ControlFreeC TSV file. It is a merge of `*_CNVs` files produced from the algorithm, and columns are [described here](http://boevalab.inf.ethz.ch/FREEC/tutorial.html#OUTPUT).
```

File: doc/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- status: available
+- changes:
+   - Updated cancer_group, adding CBTN+PNOC as PBTA, harmonized tumor_descriptor, updated clinical info for NBL samples which were missing in source files
+      - histologies.tsv  
+   - Add cnv files with PBTA+GMKF samples
+      - cnv-cnvkit.seg.gz
+      - cnv-controlfreec.tsv.gz
+      - cnv-consensus.seg.gz 
+      - consensus_seg_annotated_cn_autosomes.tsv.gz
+      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
```
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
   - Within `histologies.tsv`:
     - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket 48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
     - combine CBTN+PNOC into `cohort == PBTA`
     - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
     - updated clinical info for NBL samples which were missing in source files per [ticket 43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
   - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
      - cnv-cnvkit.seg.gz
      - cnv-controlfreec.tsv.gz
      - cnv-consensus.seg.gz 
      - consensus_seg_annotated_cn_autosomes.tsv.gz
      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
    - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
    - Add ENSG to HUGO mapping file with RMTL designation [ticket 84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket 56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```

**Review by jharenza — CHANGES_REQUESTED** [review]
Thanks @kgaonkar6 - will you update the release notes in the download per those updates?

The download otherwise works, but I would like to do one check on the histologies file later when I get a chance.

File: doc/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+     - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
+     - updated clinical info for NBL samples which were missing in source files per [ticket 43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
+   - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
+      - cnv-cnvkit.seg.gz
+      - cnv-controlfreec.tsv.gz
+      - cnv-consensus.seg.gz 
+      - consensus_seg_annotated_cn_autosomes.tsv.gz
+      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
+    - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
+    - Add ENSG to HUGO mapping file with RMTL designation [ticket 84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket 56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
- release date: 2021-06-29
- status: available
- changes:
   - Within `histologies.tsv`:
     - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
     - combine CBTN+PNOC into `cohort == PBTA`
     - harmonized tumor_descriptor per [ticket 61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
     - updated clinical info for NBL samples which were missing in source files per [ticket43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
     - updated `experimental_strategy` for targeted capture samples per [ticket62](https://github.com/PediatricOpenTargets/ticket-tracker/issues/62)
   - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
      - cnv-cnvkit.seg.gz
      - cnv-controlfreec.tsv.gz
      - cnv-consensus.seg.gz 
      - consensus_seg_annotated_cn_autosomes.tsv.gz
      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
    - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
    - Add ENSG to HUGO mapping file with RMTL designation [ticket84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```

File: doc/release-notes.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+     - updated clinical info for NBL samples which were missing in source files per [ticket43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
+     - updated `experimental_strategy` for targeted capture samples per [ticket62](https://github.com/PediatricOpenTargets/ticket-tracker/issues/62)
+   - Add cnv files with PBTA+GMKF samples per [ticket 44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
+      - cnv-cnvkit.seg.gz
+      - cnv-controlfreec.tsv.gz
+      - cnv-consensus.seg.gz 
+      - consensus_seg_annotated_cn_autosomes.tsv.gz
+      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
+    - Add EFO and MONDO cancer mapping file [ticket 78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
+    - Add ENSG to HUGO mapping file with RMTL designation [ticket84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
- release date: 2021-06-29
- status: available
- changes:
   - Within `histologies.tsv`:
     - Updated `cancer_group` logic to fix `Anaplastic` typo, remove benign tumors, and condense meningiomas [ticket48](https://github.com/PediatricOpenTargets/ticket-tracker/issues/48)
     - combine CBTN+PNOC into `cohort == PBTA`
     - harmonized tumor_descriptor per [ticket61](https://github.com/PediatricOpenTargets/ticket-tracker/issues/61) 
     - updated clinical info for NBL samples which were missing in source files per [ticket43](https://github.com/PediatricOpenTargets/ticket-tracker/issues/43)
     - add GMKF NBL ploidy per [ticket46](https://github.com/PediatricOpenTargets/ticket-tracker/issues/46)
     - updated `experimental_strategy` for targeted capture samples per [ticket62](https://github.com/PediatricOpenTargets/ticket-tracker/issues/62)
   - Add cnv files with PBTA+GMKF samples per [ticket44](https://github.com/PediatricOpenTargets/ticket-tracker/issues/44):
      - cnv-cnvkit.seg.gz
      - cnv-controlfreec.tsv.gz
      - cnv-consensus.seg.gz 
      - consensus_seg_annotated_cn_autosomes.tsv.gz
      - consensus_seg_annotated_cn_autosomes_xy.tsv.gz
    - Add EFO and MONDO cancer mapping file [ticket78](https://github.com/PediatricOpenTargets/ticket-tracker/issues/78):
    - Add ENSG to HUGO mapping file with RMTL designation [ticket84](https://github.com/PediatricOpenTargets/ticket-tracker/issues/84) and [ticket56](https://github.com/PediatricOpenTargets/ticket-tracker/issues/56)
```
just removing spacing for the links

Comment (kgaonkar6) [reply doc/release-notes.md]: Seems like the spaces are removed now

File: doc/release-notes.md (hunk 2 above)
Comment (jharenza) [reply doc/release-notes.md]: This looks good now and download works. I can't approve on my phone and for about 2 hrs, but maybe @logstar can also review and once approved, we can merge.

**Review by runjin326 — APPROVED** [review]
looks good to me!

**Comment by jharenza** [comment]
Updated the following file to match our histologies files plus updates coming in https://github.com/PediatricOpenTargets/ticket-tracker/issues/48:
```
c734c0ce531024df3f2e4eae6f1ba602  efo-mondo-map.tsv
```

**Comment by jharenza** [comment]
@kgaonkar6 I updated the release notes as well, and the md5sums for both efo and release files, and download checks out. Histologies should be the last update.

**Comment by kgaonkar6** [comment]
ok updated `histologies.tsv` , all checks passed for me.

**Review by jharenza — APPROVED** [review]
this looks ready!

**Pull request merged**